spec/system/candidate_interface/candidate_entering_personal_details_spec.rb

```diff
@@ -1,41 +1,37 @@
 require 'rails_helper'
 
 RSpec.feature 'Entering their personal details' do
-  scenario 'Logged in candidate with no personal details' do
+  scenario 'Candidate can submit their personal details' do
```

Comment:
I think we prefer to use present tense, instead of modal verbs (can/should):
`Candidate submits their personal details`